### PR TITLE
Main structure

### PR DESCRIPTION
**Pull request by NikitaBondarevv**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const image = document.getElementById('image');
+
+image.addEventListener('load', () => {
+  console.log(`Image\nWidth: ${getComputedStyle(image).width},\nHeight: ${getComputedStyle(image).height}`);
```
Comment (iliatcymbal) [inline main.js]: `getComputedStyle(image)` why do you make to times the same calculation?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "servertasks",
+  "version": "1.0.0",
+  "description": "",
+  "main": "readFromFileSystem.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "server": "node readFromFileSystem.js"
```
Comment (iliatcymbal) [inline package.json:8]: use `start` as a default command

File: readFromFileSystem.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+const http = require('http');
+const fs = require('fs');
+const imageExtebsionsReg = /\.(png|jpeg|gif)$/;
+const extebsionJsReg = /\.(js)$/;
```
Comment (iliatcymbal) [inline readFromFileSystem.js]: `extebsion` there is a mistake in the word

File: readFromFileSystem.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      if (err) {
+        response.end(errorHTML);
+        throw new Error(errorText);
+      };
+  
+      response.end(data);
+    });
+
+    return;
+  }
```
Comment (iliatcymbal) [inline readFromFileSystem.js]: looks like it is the same logic as above

File: main.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const computedStyleImage = getComputedStyle(document.getElementById('image'));
```
Comment (iliatcymbal) [inline main.js]: why do you need this?

Comment (NikitaBondarevv) [reply main.js]: To find out the size of a picture

**Review by iliatcymbal — APPROVED** [review]

**Pull request merged**